packages/api-reference/src/features/sidebar/components/SidebarElement.vue

```diff
@@ -127,7 +139,14 @@ const onAnchorClick = async (ev: Event) => {
         :tabindex="hasChildren ? -1 : 0"
         @click="onAnchorClick">
         <p class="sidebar-heading-link-title">
-          {{ item.title }}
+          <span
+            v-if="config.operationTitleSource === 'path'"
```

Comment:
we should use a computed that has the if logic, and we might even want to have this logic in the traverse functions cc @amrit

Reply:
yes, just display